Quake/common.c

```diff
@@ -2189,6 +2190,14 @@ static void COM_Game_f (void)
 			}
 		}
 
+		q_strlcpy (newgamedir, va("%s/%s", com_basedir, p), sizeof(newgamedir));
```

Comment:
This extra copy to a newgamedir doesn't seem necessary: what is stopping you from feeding va(...) directly to Sys_FileType?

Reply:
Thanks for reviewing @sezero. You're right. I think I initially used that for debugging purposes, but forgot about the fact that it wasn't necessary afterwards. Since it serves no other purpose, I'll remove it from the patch.